package.json

```diff
@@ -139,13 +141,16 @@
     "merge-options": "^1.0.1",
     "mime-types": "^2.1.21",
     "mkdirp": "~0.5.1",
+    "mortice": "^1.2.2",
     "multiaddr": "^6.0.5",
     "multiaddr-to-uri": "^4.0.1",
     "multibase": "~0.6.0",
     "multicodec": "~0.5.1",
     "multihashes": "~0.4.14",
     "multihashing-async": "~0.6.0",
+    "nanoid": "^2.0.3",
```

Comment:
We already use `hat`, does `nanoid` add anything that `hat` doesn't provide?

Reply:
Doesn't rely on `Math`